docs/guides/install/index.html

```diff
@@ -78,40 +77,6 @@ <h2 class="hxSectionTitle" id="web-components">Web Components</h2>
     </p>
   </section>
 
-  <section><!-- TL;DR -->
```

Comment:
Removed section. It was repetitive and is essentially replaced with the new layout templates.